src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -315,5 +309,17 @@ public boolean isChassisMoving()
     }
 
   }
-
+  public Boolean isSwerveNotSynched() {
```

Comment:
Ideally use boolean not Boolean. boolean is a primitive data type whereas Boolean is a class.